nessusToExcel.py

```diff
@@ -1795,6 +1878,7 @@ def CloseExcelWriter(writer):
 parser.add_argument('--quiet', '-q', action='store_true', help='Accept defaults during execution')
 parser.add_argument('--keyword', '-k', required=False, help='Extract all information relating to this word')
 parser.add_argument('--noclean', '-n', required=False, action='store_true', help='Do not remove duplicates and merge columns in outdatedsoftware')
+# parser.add_argument('--nostyle', '-s', required=False, action='store_true', help='Do not apply any style')
```

Comment:
Do you want this commented out?

Reply:
Not yet, it's not implemented. I didn't have enough time to get to that point. Anyway if the user leaves the json file empty, most of the style is not applied anyway. I wanted to add that flag so no style at all is applied.